.github/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
 
 ## 0.7.0
+1. [MISC] Added JP, FI, SE, NO and NL language support for tooltips. - @kpotschi (MelodyConnor)
```

Comment:
**suggestion**: Move this to the bottom of the list, as the newest items go there. Think of it as a book 😀

Reply:
got it.

Reply:
This conversation is marked as resolved, btu the changelog entry is still at the top 😀.

Reply:
oh, so i gotta make a new commit based on the comments? I somehow thought resolving here would "merge" it, low-key. but well, that's the learning curve I'm here for :)